ansible/templates/whisk.properties.j2

```diff
@@ -35,6 +35,9 @@ whisk.api.vanity.subdomain.parts=1
 
 runtimes.manifest={{ runtimesManifest | to_json }}
 
+prewarm.kind={{ prewarmKind | default('nodejs:6') }}
```

Comment:
don't think we'll need these until they play into a test which shouldn't be the case.

Reply:
@markusthoemmes should we eschew these?